src/App.js

```diff
@@ -0,0 +1,28 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import BooksList from './components/BooksList';
+import BookForm from './components/BookForm';
+import Navigation from './components/Navigation';
+
+const HomePage = () => (
+  <div>
+    <h1>Home Page</h1>
+    <BooksList />
+    <BookForm />
+  </div>
+);
+
+const CategoriesPage = () => <h1>Categories Page</h1>;
```

Comment:
- [ ] Kindly make sure the `categories` is a component which should be imported here

### Instructions:
![image](https://github.com/CVILLA09/Bookstore/assets/50990925/1e683fa7-998b-4528-855d-f7b13b45edf9)